public/about-team.html

```diff
@@ -175,41 +175,47 @@ <h4>Director</h4>
               <h2>Administrative</h2>
             </div>
             <div class="team-container">
-              <div class="team-lead-card" id="5c227d8b0570397fe37636e3">
+              <div class="team-lead-card" id="5c227d8b0570397fe37636e4">
+                <div class="team-lead-img">
+                  <img
+                    src="images/teamleads/Sebastian_Thompson_President.jpeg"
+                  />
+                </div>
+                <div class="team-lead-name">President</div>
+                <div class="team-lead-text">Sebastian Thompson</div>
+                <div class="team-lead-text">Mechanical Engineering</div>
+                <div class="team-lead-text">Senior</div>
+              </div>
+              <div class="team-lead-card" id="5c227d8b2570397fe37636e6">
                 <div class="team-lead-img">
-                  <img src="images/teamleads/JacobH.jpg" />
+                  <img src="images/teamleads/placeholder.jpg" />
                 </div>
                 <div class="team-lead-name">Electrical Director</div>
-                <div class="team-lead-text">Jacob Hogan</div>
-                <div class="team-lead-text">Electrical Engineering</div>
+                <div class="team-lead-text">Ryan Bear</div>
+                <div class="team-lead-text">Electrical Engineering and CS</div>
                 <div class="team-lead-text">Senior</div>
               </div>
-              <div class="team-lead-card" id="5da53d9b70eb0024efca2eef">
+              <div class="team-lead-card" id="5c227d8b2570397fe37636e6">
                 <div class="team-lead-img">
-                  <img src="images/teamleads/Jay Yoo_Chassis and Braking.jpg" />
+                  <img src="images/teamleads/placeholder.jpg" />
                 </div>
                 <div class="team-lead-name">Mechanical Director</div>
-                <div class="team-lead-text">Jay Yoo</div>
```

Comment:
Jay Yoo is the Mechanical director, some of the form responses are probably old if that's what you were going off of. Also Jacob Hogan is still the electrical director. All of the people/roles should be the same as what is currently on the site except for Mridhul, Shriya, and Kyle